credsweeper/ml_model/features.py

```diff
@@ -22,6 +22,7 @@ def __call__(self, candidates: List[Candidate]) -> List[bool]:
             candidates: list of candidates to extract features
 
         """
+        # pylint: disable=E1101
```

Comment:
In this case i think it would be better to add abstract method to the `Feature` class
Similar to 
```python
    @abstractmethod
    def extract(self, candidate: Candidate) -> Any:
        raise NotImplementedError
``` 

In this case while all subclasses do have `extract` in them, it is really missing from the base abstract class, and should be added


We might do it with a different PR, it's just something I want to point out

Reply:
+